File: app/src/main/java/com/ios7/vibeify/MyClasses/EzTimer.java
package com.ios7.vibeify.MyClasses;

import android.os.Handler;
import android.os.Looper;

public class EzTimer {

    private static final Handler handler = new Handler(Looper.getMainLooper());

    // Method to run code with delay
    public static void runWithDelay(long delayMillis, Runnable action) {
        handler.postDelayed(action, delayMillis);
    }
}

File: app/src/main/java/com/ios7/vibeify/MyClasses/EzTimer.kt
package com.ios7.vibeify.MyClasses

import android.os.Handler
import android.os.Looper

object EzTimer {
    private val handler = Handler(Looper.getMainLooper())

    @JvmStatic
    fun runWithDelay(delayMillis: Long, action: Runnable) {
        handler.postDelayed(action, delayMillis)
    }
}
